src/System.IO.Pipelines/tests/Performance/System.IO.Pipelines.Performance.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IncludePerformanceTests>true</IncludePerformanceTests>
+    <ProjectGuid>{66AE57BA-B56C-4A1E-ACA6-7C18431D416B}</ProjectGuid>
+    <Configurations>netstandard-Debug;netstandard-Release</Configurations>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(CommonTestPath)\System\Diagnostics\RemoteExecutorConsoleApp\RemoteExecutorConsoleApp.csproj">
+      <Project>{69e46a6f-9966-45a5-8945-2559fe337827}</Project>
+      <Name>RemoteExecutorConsoleApp</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\System.IO.Pipelines.Tests.csproj">
```

Comment:
I am assuming this is needed for TestMemoryPool? Maybe include that file explicitly and not the whole test project?

Reply:
Yes, `TestMemoryPool` is in `System.IO.Pipelines.Tests`. But it looks like that I could use the default `MemoryPool`. I will remove the dependency.